### DISCUSSION TITLE
Dashboard Filter and highlight

### DISCUSSION THREAD

**Discussion by lkNGAT** [scalar/scalar#752] [Ideas]
When having a lot of different score names the line chart gets really crowded. 
Would it be possible to add filtering, similar to the bahaviour on the other pages?

Filtering for models might be difficult (at least to be meaningful) since traces can consist of different model calls, while scores can get assigned to the whole trace.

It would also be great to be able to click on a score name (in the legend) and the score would get highlighted.

**Comment by marclave** [comment]
hey @lkNGAT , do you mean the models at the bottom of the rendered doc?

Would you be able to share your OAS/Swagger spec? :)

**Comment by marcklingen** [comment]
Sorry for the confusion @marclave, this idea is regarding https://github.com/langfuse/langfuse

I loved how you routed q&a and ideas to Discussions ([config.yml](https://github.com/scalar/scalar/blob/main/.github/ISSUE_TEMPLATE/config.yml)) and was a bit too quick when adding it to our repo without replacing the links accordingly. This was fixed today ([PR](https://github.com/langfuse/langfuse/pull/867)).

Scalar seems to be great, impressed by the UX and DX. We will move the Langfuse api ref from redoc to scalar once there is support for models in responses ([idea](https://github.com/scalar/scalar/discussions/645)).

@lkNGAT can you move this idea to the langfuse repo?

**Reply by lkNGAT** [reply]
Hi @marclave 
I am also sorry for the confusion.
As @marcklingen already mentioned I unintentionally created this issue here.
I will close it right now.
@marcklingen Already did.
